default/stringtables/es.txt

```diff
@@ -6213,8 +6213,8 @@ Guía de Inicio Rápido
 #* <not translated 1970-01-01 01:00:00>
 #* <not translated 1970-01-01 01:00:00>
 
-#*RESEARCH_TECH_GUIDE_TITLE
-#* <not translated 1970-01-01 01:00:00>
+RESEARCH_TECH_GUIDE_TITLE
+Desarrollo Tecnológico
```

Comment:
Research has been translated as _Investigación_ in several places. So I prefer "Investigando tecnologías" (or, if you prefer, "Investigación Tecnológica", but I think the other is better).

Reply:
I settled for "Investigación Tecnológica" it was more fitting to the already present translations that linked to this string and I also think is more fitting for a page title.